cli/src/config.rs

```diff
@@ -38,9 +41,14 @@ fn load(path: &PathBuf) -> Result<CliConfig, Box<dyn Error + 'static>> {
         CliConfig::create(path)?;
     }
 
+    let mut s_defaults = Config::new();
```

Comment:
It solves the problem with environment variable override nicely. Allowing us to declare a namespace `APOLLO_` and automatically generating the rest of the environment variables from there instead of having to create new ones.
Having to hardcode the `api_url` string isn't ideal but I think using the `Config` abstraction does provide benefits.